### PR TITLE
Add CFLAGS to setup.

### PR DESCRIPTION
**Pull request by iemelyanov**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -25,13 +26,19 @@
     else:
         macros = []
 
+    CFLAGS = ['-O2']
+    # CFLAGS = ['-g']
+    if platform.uname().system != 'Windows':
```
Comment (webknjaz) [inline setup.py]: `platform.platform()`

Comment (iemelyanov) [reply setup.py]: platform.platform() gives long string with lots of additional information, not just system name

Comment (webknjaz) [reply setup.py]: Oh, I meant `platform.system()`

Comment (iemelyanov) [reply setup.py]: Ok, thanks)

**Pull request merged**